common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/FederationBatchStatus.java

```diff
@@ -2,5 +2,6 @@
 
 public enum FederationBatchStatus {
```

Comment:
Just a suggestion to add a description of each status code and what it represents.